JetNews/app/src/main/java/com/example/jetnews/ui/interests/SelectTopicButton.kt

```diff
@@ -52,7 +49,10 @@ private fun SelectTopicButtonOn(modifier: Modifier = Modifier) {
         shape = CircleShape,
         modifier = modifier
     ) {
-        Icon(vectorResource(R.drawable.ic_check))
+        Icon(
+            asset = vectorResource(R.drawable.ic_check),
+            tint = MaterialTheme.colors.onPrimary
```

Comment:
Snuck in a bug fix revealed by dark theme.

Reply:
Should this have an emphasis?

Reply:
Yeah, good catch. Should be high emphasis. Done.